inc/api/endpoints/controller/class-posts.php

```diff
@@ -94,7 +94,7 @@ public function overrideResponse( $response, $post, $request ) {
 			}
 		}
 
-		if ( $post->post_type === 'part' ) {
+		if ( $post->post_type === 'part' && isset( $response->data['meta'] ) ) {
```

Comment:
Sometimes we don't have the `meta` key in the API response, for example in the `toc` endpoint in the part section `meta` is not exposed.